x-pack/plugins/cases/server/routes/api/types.ts

```diff
@@ -7,23 +7,10 @@
 
 import type { Logger } from 'kibana/server';
 
-import type {
-  CaseConfigureService,
-  CaseService,
-  CaseUserActionService,
-  ConnectorMappingsService,
-  AttachmentService,
-} from '../../services';
-
 import type { CasesRouter } from '../../types';
 
 export interface RouteDeps {
-  caseConfigureService: CaseConfigureService;
```

Comment:
The HTTP routes don't need the services since they just access the cases client now.